src/app/cube/details/reviewer-panel/reviewer-panel.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, OnInit, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'clark-reviewer-panel',
+  templateUrl: './reviewer-panel.component.html',
+  styleUrls: ['./reviewer-panel.component.scss']
+})
+export class ReviewerPanelComponent implements OnInit {
+  @Input() releasedDate;
+  @Input() revisedDate;
+  @Output() download: EventEmitter<boolean> = new EventEmitter();
+  constructor() { }
+
+  ngOnInit() {
+  }
+  click(download: boolean) {
```

Comment:
Can you give this function a more descriptive name?

Reply:
Resolved with Change click to downloadRevisions.